### DISCUSSION TITLE
Elsa2 dashboard as a separate server errored

### DISCUSSION THREAD

**Discussion by SachinMehndiratta** [elsa-workflows/elsa-core#951] [Q&A]
Hi,

Tried to implement the Elsa2 dashboard as a separate server with Blazor runtime model set as Server throws error on few calls.

![image](https://user-images.githubusercontent.com/29795134/117577052-cd2a8200-b105-11eb-92a0-3c5475866e23.png)

Not sure if I am doing something wrong here.

**Comment by sfmskywalker** [comment]
Make sure to build the client-side assets (including the stuff from the designer/elsa-workflows-studio and the bindings project) using npm install and then npm run build.

The AppVeyor build does this automatically which is why the packages do contain these files. If you clone the source code then this is something one must do manually.

Other than that, I recommend using the ASP.NET Core dashboard sample project instead of the Blazor one. The former one is simpler yet is more feature complete. Blazor will be updated at some point, but later.
